front/components/assistant/RenderMessageMarkdown.tsx

```diff
@@ -420,7 +455,7 @@ function LinkBlock({
       href={href}
       target="_blank"
       rel="noopener noreferrer"
-      className="break-all text-blue-500 hover:underline"
+      className="transition: all 0.2s break-all font-semibold text-action-500 ease-in-out hover:text-action-400 hover:underline active:text-action-600"
```

Comment:
I don't think that works @PopDaph !
This is CSS and we want classNames

Reply:
transition-all duration-200 probably

Reply:
Added, thanks!